### PR TITLE
Added three places to eat in portland

### PR DESCRIPTION
**Pull request by rsnakamura**

### CODE REVIEW

**Comment by jpallepo** [comment]
would like to visit and eat there

File: portland-eats.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Places to eat in Portland, OR
+
+- My kitchen (limited seating)
```
Comment (tintaton) [inline portland-eats.md]: Maybe someday?

File: portland-eats.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+- Tangier
```
Comment (tintaton) [inline portland-eats.md]: What kind of food is this?

**Comment by Estherbunny** [comment]
do any of these places have good craft beer?

**Comment by Estherbunny** [comment]
the HUB looks awesome - hopefully one day i will make it to Portlandia

**Pull request merged**